xsnap/makefiles/lin/xsnap-worker.mk

```diff
@@ -31,7 +31,6 @@ C_OPTIONS = \
 	-DmxLockdown=1 \
 	-DmxMetering=1 \
 	-DmxDebug=1 \
-	-UmxInstrument \
```

Comment:
Commit says "mxInstrument must be undefined to disable", but isn't that what `-U` makes explicit?

Reply:
I just misread the -U as another -D.